community/releases/versions.md

```diff
@@ -31,11 +31,14 @@ Released: {{ post.date | date_to_rfc822 }}
   {% endunless %}
 {% endfor %}
 
-### Deprecated Versions
+## Deprecated Versions
 {% for post in reversed %}
   {% if post.tags contains 'deprecated' %}
 #### {{ post.title }}  
 Released: {{ post.date }}  
 <a href="{{ post.url }}">Changelog</a>
   {% endif %}
 {% endfor %}
+
+> To be notified when new Spinnaker versioned releases are available, please join the
+[spinnaker-announce](https://groups.google.com/forum/#!forum/spinnaker-announce) group.
```

Comment:
Maybe note that you will first need to sign in with a google account in order to join? Otherwise we'll get questions on how to join via that page.